tasks/__init__.py

```diff
@@ -140,7 +153,7 @@ def i18n(ctx, update=False):
 
     info('Extract Python strings')
     with ctx.cd(ROOT):
-        ctx.run('python setup.py extract_messages')
+        ctx.run(f"pybabel extract . --mapping-file={EXTRACT_I18N_MAPPING_FILE} --output-file={EXTRACT_I18N_OUTPUT_FILE} --width={EXTRACT_I18N_WIDTH}")
```

Comment:
Could we migrate infos that were in setup.cfg to pyproject?

Infos that are in babel.cfg don't seem use, ignoring templates.

Reply:
pybabel doesn't seem to support `pyproject.toml` at this time :/

Reply:
And we have to use the pybabel CLI which seems to need the info as arguments